doc/source/deploy-on-yarn.rst

```diff
@@ -98,19 +99,13 @@ Start by activating a pre-existing environment for dependency management.
 
 .. code-block:: bash
 
-    source /home/rayonyarn/miniconda3/bin/activate
+    source environment/bin/activate
 
-Obtain the Skein Application ID which is used when pushing addresses to worker services.
+Register the Ray head addresses needed by the workers in the Skein key-value store.
```

Comment:
Is there more than one address? Otherwise you might want "address" instead of "addresses".